resources/assets/sass/_footer.scss

```diff
@@ -0,0 +1,19 @@
+footer {
```

Comment:
I prefer avoiding using `header` and `footer` for CSS hooks; they're not intended to be once-per-page type elements, so I prefer something like `.page-footer`.

And the question applies here, too: Is there any benefit of pulling these classes out of the DOM?